api/models/comment/createAndPresentComment.js

```diff
@@ -106,3 +107,10 @@ const notifySockets = (comment, post) => {
     return post.pushCommentToSockets(comment)
   }
 }
+
+const createOrUpdateConnections = (userId, existingFollowers) => comment => {
+  // Deliberately non-blocking (don't wait for promise to resolve/reject)
+  existingFollowers
+    .filter(f => f !== userId)
+    .forEach(follower => UserConnection.createOrUpdate(userId, follower, 'message'))
+}
```

Comment:
@levity Is this ok? I didn't want to wait for the promise (hence not returning it to the `.tap`, which will only block if promise is returned) since there might be quite a few queries involved in a large thread. However, if I do this I should probably log errors somewhere with a `.catch`... how do we handle logging?